apps/L1G/restaurant/restaurant-frontend/src/components/auth/update-user-profile/ProfilePictureUpload.tsx

```
@@ -0,0 +1,57 @@
+'use client';
+import type React from 'react';
+import { useState } from 'react';
+import { Button } from '@/components/ui/button';
+import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar';
+import { Edit, User } from 'lucide-react';
+import { uploadCloudinaryPicture } from '@/utils/ProfilePicCloudinary';
+
+interface ProfilePictureUploadProps {
+  currentImage?: string;
+  onImageUpdate: (imageUrl: string) => void;
```

ESLint

[error] 'imageUrl' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)